Readme.md

```diff
@@ -105,19 +106,19 @@ Good manners:
   "Implement EZP-201xx: Add support for X in Y"
```

Comment:
This is very outdated. My fault :)
In short, we don't give orders what to do to our commits ;) We say what we did ;) 

``` Markdown
  "EZP-20104: Fixed ContentController to return error status when content is not found"
  or
  "EZP-20105: Added support for X in Y"
```